### DISCUSSION TITLE
View Uploads from Other Users

### DISCUSSION THREAD

**Discussion by FuckingToasters** [diced/zipline#474] [Questions] [answered]
Hello, I as the Administrator of the Website like to know what type of Images others Upload, I want to have control over User's Content so in case they upload bad things like Child Porn, i can act on it appropiatly.

Regards,
FuckingToasters

**Comment by diced** [comment, marked as answer]
If you are an administrator you can view other users uploads from the users page (on the sidebar, its under Administration).

**Reply by xxvolt** [reply]
sorry to revive this old thread, on the users page I dont see any options to see other's uploads, just the users and the ability to change their password, maybe there is something to add in the docker compose? idk

**Reply by diced** [reply]
> sorry to revive this old thread, on the users page I dont see any options to see other's uploads, just the users and the ability to change their password, maybe there is something to add in the docker compose? idk

sorry it's not really labelled correctly 😂 understandable that you didn't see
![image](https://github.com/user-attachments/assets/377d8aef-470a-45b2-b6cd-12e8132dcc1c)
